test/actions.js

```diff
@@ -100,6 +117,27 @@ module.exports = (app, template, hook) => {
           });
       });
 
+      it('Create a Form 2 for Action tests', (done) => {
+        request(app)
+          .post(hook.alter('url', '/form', template))
+          .set('x-jwt-token', template.users.admin.token)
+          .send(tempForm2)
+          .expect('Content-Type', /json/)
+          .expect(201)
+          .end((err, res) => {
+            if (err) {
+              return done(err);
+            }
+
+            tempForm2 = res.body;
+
+            // Store the JWT for future API calls.
+            template.users.admin.token = res.headers['x-jwt-token'];
+
+            done();
+          });
+      })
+
```

Comment:
When you are setting up data for a test, it should be either be in a before function or inline with the test itself.  When you put inside a test it creates dependency where you have to run one test before another.

Reply:
combined into one test